app/styles/ui/_tutorial.scss

```diff
@@ -1,12 +1,117 @@
 .tutorial-panel-component {
+  display: flex;
+  flex-flow: column nowrap;
   min-width: 300px;
+  max-width: 350px;
+  background-color: var(--background-color);
+  box-shadow: var(--base-box-shadow);
+  z-index: var(--side-panel-z-index);
+  overflow-y: scroll;
+
+  .titleArea {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: var(--spacing-triple) var(--spacing-double) var(--spacing-double) var(--spacing-quint);
+
+    h1 {
+      margin: 0;
+      font-weight: var(--font-weight-light);
+    }
+    svg {
+      height: 55px;
+      width: 55px;
```

Comment:
Looking at the SVG it seems the viewport is actually 56x56. We should either update that here so that we stick to a 1:1 pixel ratio in the SVG or we can do what I propose below and tighten up the SVG, removing the white space around it so that we can position it better (in which case I believe it'll be 44x44)